pages/locations/bengaluru/index.md

```diff
@@ -1,7 +1,9 @@
 ---
-title: Bangalore
+# The title should just be the city's name and must also match the city name as it
+# appears in the SmartRecruiters data used for the job listings.
+title: Bengaluru
 # URI slug for this location which gets appended to https://buildit.wiprodigital.com/thing/studio/[xx]/
-citySlug: bengaluru
+citySlug: bengaluru 
```

Comment:
OMG THIS LINE ENDS WITH WHITESPACE!!!!11

Reply:
Uh oh! @ashblue's gonna kill me!

Reply:
Get a frontmatter linter in here asap! 🤔